Jenkinsfile

```diff
@@ -77,6 +76,27 @@ Map nlt_test() {
     Date startDate = new Date()
     sh label: 'Fault injection testing using NLT',
        script: './ci/docker_nlt.sh --class-name el8.fault-injection fi'
+    List filesList = []
+    filesList.addAll(findFiles(glob: '*.memcheck.xml'))
+    int vgfail = 0
+    int vgerr = 0
+    if (filesList) {
+        String rcs = sh label: 'Check for Valgrind errors',
+               script: "grep -E '<error( |>)' ${filesList.join(' ')} || true",
```

Comment:
I know there's been a lot of discussion on the topic of desired CI failure in the face of valgrind warnings which I haven't been following but I really dislike the idea of using grep on xml files and I still don't grasp what is so unique about our configuration that means we need to.  Using valgrind with Jenkins is a well-trodden path that just seems to be really tricky for us.

Reply:
The desire here is to get truth in the display of the color of the Jenkins step/stage where an issue was detected to improve debugging issues and stage reporting to Github.

The publishValgrind plugin can not be used in parallel steps, so it must be called only once at post section for the entire pipeline.  An undocumented restriction that was discovered by us.

So grep of the valgrind XML files is the easiest way to get an indication on the stage where a valgrind issue was found, which is what we have been doing for Junit XML files for a while.